packages/apps-v5/README.md

```diff
@@ -13,7 +13,7 @@ $ npm install -g @heroku-cli/plugin-apps-v5
 $ heroku COMMAND
 running command...
 $ heroku (-v|--version|version)
-@heroku-cli/plugin-apps-v5/7.68.0 darwin-x64 node-v16.19.0
+@heroku-cli/plugin-apps-v5/7.68.1 darwin-arm64 node-v16.16.0
```

Comment:
Was the node version intentionally dropped here?

Reply:
Oh that's interesting... that must have happened in the script, but I'm not sure why...

Reply:
Oof. I wonder if that's because I am using node 16.16 locally, instead of 16.19.

Reply:
Oooh, that would make sense. In the future, we should update parts of that script because sometimes weird discrepancies like this occur. I too have run into my fair share 😝

Reply:
I fixed it manually. Thanks for catching it!

Reply:
I'm making a ticket to investigate, but I think we can proceed with the release.